central/notifiers/splunk/splunk.go

```diff
@@ -203,9 +203,6 @@ func (s *splunk) getHTTPToken() (string, error) {
 		return s.creds, nil
 	}
 
-	if s.GetNotifierSecret() == "" {
```

Comment:
Agreed on removal since it was never checking originally. I guess it's possible to set up a local splunk with no auth